react/features/app/components/App.native.js

```diff
@@ -96,6 +104,12 @@ export class App extends AbstractApp {
     componentDidMount() {
         super.componentDidMount();
 
+        this._init.then(() => {
+            // We set the color scheme early enough so then we avoid any
+            // unnecessary re-renders.
+            this.state.store.dispatch(setColorScheme(this.props.colorScheme));
```

Comment:
Should, yes. This is the same way we delay app init for the time we create the store and rehydrate the values from localStorage. Can't think of anything earlier.

Reply:
I ask because technically we are chaining to after the state has been initialized, so I thought it might be possible to start seeing the old theme and a swap.